helper/testhelpers/testhelpers_oss.go

```diff
@@ -20,3 +20,28 @@ func WaitForActiveNodeAndStandbys(t testing.T, cluster *vault.TestCluster) {
 		}
 	}
 }
+
+// WaitForActiveNodeAndSelectedStandbys is variation on WaitForActiveNodeAndStandbys,
+// the difference is that this function can be given core indexes which it should not attempt to wait for.
+func WaitForActiveNodeAndSelectedStandbys(t testing.T, cluster *vault.TestCluster, indexesToSkip ...int) {
```

Comment:
🤔 The name here implies to me that you're waiting for the active node and also waiting on the selected standbys, whereas the behavior is the opposite (you're not waiting on standbys). Maybe `WaitForActiveNodeButNotGivenStandbys`? 🤷